content/concepts/basics/04-search.md

```diff
@@ -4,17 +4,37 @@ metaTitle: "sensenet - Search"
 metaDescription: "sensenet Search"
 ---
 
-# Search
-https://community.sensenet.com/docs/search-engine/
-https://wiki.sensenet.com/query_syntax.html
-# Full text search
+
+In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the ui is there because a query found it. In this article we describe the search module and how developers can customize it.
+
 # Metadata indexing, searching by fields
-https://wiki.sensenet.com/Field_Indexing.html
+For every content [link] the field values can be indexed so that when searched for a field value the corresponding content will appear in the result set. It is also possible to search in fields by explicitly defining the field whose values are to be searched within a query. The way a specific field of a content is indexed is defined in the CTD Field Definition.
+
+sensenet uses the Lucene search engine for indexing the content repository and to provide a fast mechanism for returning query results. Apart from the indexing of some basic built-in properties every field can be configured to be indexed separately.
+
+You may configure which search engine you want to use for indexing and querying by setting the SearchEngine value in the providers web.config section.
+Please note that this decision is usually made in development time. Although it is possible to switch to another search engine on a portal later, it will involve reindexing the whole content repository.
```

Comment:
```suggestion
Please note that this decision is usually made in development time. Although it is possible to switch to another search engine later, it will involve reindexing the whole content repository.
```

Reply:
ezt a web.configos részt inkább full kiszedném, sztem itt too much